### ISSUE TITLE
Maraudon - Spewed Larva

### ISSUE THREAD

**Issue by scotty0100** [classicdb/database#772]
The GO (178559) Larva Spewer don't seem to be functioning properly. They don't spew out Larva!

Nor when right clicking on them to 'Kill' them, they just stay there.

**Comment by cala** [comment]
Thank you. It was already on my TODO list after the recent Maraudon rework, but it is good to have something to track it.

**Comment by Lumber-Jack-2** [comment]
just found this: http://www.wowhead.com/npc=13533/spewed-larva

maybe the go should spawn periodically these npcs?

**Comment by Lumber-Jack-2** [comment]
confirmed from sniffs:
INSERT INTO `gameobject_template` (`entry`, `size`, `faction`, `flags`) VALUES
(178559, 0.805835, 14, 0), -- 178559

strange go :(

``` sql

ServerToClient: SMSG_QUERY_GAME_OBJECT_RESPONSE (0x005F) Length: 155 ConnIdx: 0 Time: 03/04/2010 01:23:38.000 Number: 12693
Entry: 178559
Type: Trap (6)
Display ID: 3912
[0] Name: Larva Spewer
[1] Name: 
[2] Name: 
[3] Name: 
Icon Name: 
Cast Caption: 
Unk String: 
[0] Data: 99
[1] Data: 0
[2] Data: 0
[3] Data: 0
[4] Data: 0
[5] Data: 0
[6] Data: -1000
[7] Data: 0
[8] Data: 0
[9] Data: 0
[10] Data: 0
[11] Data: 0
[12] Data: 0
[13] Data: 0
[14] Data: 0
[15] Data: 0
[16] Data: 0
[17] Data: 0
[18] Data: 0
[19] Data: 0
[20] Data: 0
[21] Data: 0
[22] Data: 0
[23] Data: 0
Size: 0.805835
[0] Quest Item: 0 (0)
[1] Quest Item: 0 (0)
[2] Quest Item: 0 (0)
[3] Quest Item: 0 (0)
[4] Quest Item: 0 (0)
[5] Quest Item: 0 (0)
```

**Comment by evil-at-wow** [comment]
Yes, I can confirm the GO should periodically spawn a mob that's using waypoints. When you get near the GO, you'll almost certainly run into them. Clicking the GO stops the spawning.

**Comment by cala** [comment]
Thanks for those. My question is : how using the GO does stop the spawning? Is there some kind of spell cast? Or is this just a script on go use (or something similar)? Do you have some data on this, please ?

**Comment by evil-at-wow** [comment]
If my memory serves me well, using the GO 'destroys' it (visually). It's a type 6 (Trap), so I guess that just disables the trap? I don't know for sure what exactly happens though.

I had a quick scan in my data, to find sessions with this GO. For those, I then searched every occurrence of either the GUID itself or its packed version, assuming it would show up in spell casts or other uses. Apart from the packet telling the client about the spawn, I only found one SMSG_UPDATE_OBJECT packet (changing the GO's dynamic flags to 0x00640000) and one SMSG_GAMEOBJECT_CUSTOM_ANIM packet when using it. I'm not sure if that tells us anything...
